ecdsa/signing/rounds.go

```diff
@@ -147,6 +147,27 @@ func (round *base) sendMessage(msg tss.ParsedMessage) *tss.Error {
 	}
 }
 
+func (round *base) sendSignature() {
+	// shouldn't ever reach these cases, but it is better to be safe than sorry.
+	if round.out == nil {
+		return
+	}
+	if round.Params() == nil {
+		return
+	}
+
+	if round.Params().Context == nil {
+		round.end <- round.data
+		return
+	}
+
+	select {
+	case round.end <- round.data:
```

Comment:
should we also add a timeout here?

Reply:
No, I think the outputs of the fullparty should always be listened to.